src/mgr/StandbyPyModules.cc

```diff
@@ -72,29 +75,30 @@ void StandbyPyModules::shutdown()
   modules.clear();
 }
 
-int StandbyPyModules::start_one(PyModuleRef py_module)
+void StandbyPyModules::start_one(PyModuleRef py_module)
 {
   std::lock_guard l(lock);
-  const std::string &module_name = py_module->get_name();
-
-  ceph_assert(modules.count(module_name) == 0);
-
-  modules[module_name].reset(new StandbyPyModule(
-      state,
-      py_module, clog));
-
-  int r = modules[module_name]->load();
-  if (r != 0) {
-    modules.erase(module_name);
-    return r;
-  } else {
-    dout(4) << "Starting thread for " << module_name << dendl;
-    // Giving Thread the module's module_name member as its
-    // char* thread name: thread must not outlive module class lifetime.
-    modules[module_name]->thread.create(
-        modules[module_name]->get_name().c_str());
-    return 0;
-  }
+  auto name = py_module->get_name();
+
+  ceph_assert(modules.count(name) == 0);
+
+  modules[name].reset(new StandbyPyModule(state, py_module, clog));
+  auto standby_module = modules.at(name).get();
```

Comment:
And why using `[]` instead of `.at()`. The advantage of `at` that it will not try to insert key if it does not exist, thought it is not relevant here. The main rationale was to have the code similar to what we have for ActivePyModules.

Reply:
okay. it's just weird to do so. as we start worrying about what if it does not exist after inserting an item in the map.